pkg/systemd/unit_test.go

```diff
@@ -119,35 +121,76 @@ managedUnits:
 	}
 }
 
-type NoopEnsureControl struct{}
+type NoopEnsureControl struct {
```

Comment:
with state the `Noop` name is no longer fitting, it's more a "fake" now

Reply:
I was considering it but I think noop is still a better fit since Enable/Start methods don't affect the state

Reply:
But that will be a good point when we extend it to handle active states differently and have to test it